### DISCUSSION TITLE
A reliable transport for Azure Servie Bus

### DISCUSSION THREAD

**Discussion by penderi** [BrighterCommand/Brighter#1437] [Q&A]
Is there a ASB transport available for brighter?
 Or more accurately, is there an aoffering as mature as SQS/rabbit?

At Flagstone we use brighter. We are hosted on Azure and lightly use ASB. Weare about to distribute and as such are looking to use brighter to distribute to ASB topics.

We know there are a few Poc/similar in usage (from @domspektrix and @dstockhammer ). This seems like a good opportunity to:
1 - collect and PR what we have
2 - produce a list of missing features/issues
3 - Flagstone to contribute / see where we are.

We've agreement for a few stories over the coming sprints to contribute. We're asking for help on points 1 & 2 so that together we can give brighter a good boost in this space!

**Comment by preardon** [comment]
@penderi Would be happy to lend a hand mate
